elements/pl-code/pl-code.py

```diff
@@ -131,6 +155,14 @@ def prepare(element_html, data):
                 f'Unknown language: "{language}". Must be one of {", ".join(allowed_languages)}'
             )
 
+    style = pl.get_string_attrib(element, "style", STYLE_DEFAULT)
+    pygments_style = get_style_by_name(style)
+    if pygments_style is None:
+        allowed_styles = STYLE_MAP.keys()
+        raise Exception(
```

Comment:
```suggestion
        raise KeyError(
```

Reply:
Replaced all generic Exceptions with `KeyError` or `ValueError` in 118f90313.